core/controllers/topic_editor_test.py

```diff
@@ -261,10 +285,26 @@ def test_editable_topic_handler_put(self):
             }, {
                 'change_affects_subtopic_page': True,
                 'cmd': 'update_subtopic_page_property',
-                'property_name': 'html_data',
-                'old_value': '',
+                'property_name': 'page_contents',
```

Comment:
Are we going to handle the 'subtitle_html' and 'content_ids_to_audio_translations' with a single command?

Like in exploration we have two separate property `STATE_PROPERTY_CONTENT` and `        STATE_PROPERTY_CONTENT_IDS_TO_AUDIO_TRANSLATIONS`. 

The editor can apply changes to both properties (in two different commands) where as a translator can only apply to the latter one. 

How are we going to handle translator command?

Reply:
done